docs/DeveloperGuide.md

```diff
@@ -476,7 +555,8 @@ concrete predicate classes were implemented:
 5. `FilterPropsByOwnerNamePredicate`
 
 Based on command parameters passed in by the user, these predicates are constructed and composed together to form a single
-`Predicate`, which is then used to filter the `ObservableArrayList` directly.
+`Predicate`, which is then used to filter the `ObservableArrayList` directly. More specifics regarding composition behaviour
```

Comment:
Might want to replace "composition behaviour" as I thought you meant the whole-part relationship kind of composition